app/src/editor/canvas/components/Ports/Port/index.jsx

```diff
@@ -193,9 +192,4 @@ const Port = ({
     )
 }
 
-// $FlowFixMe
-const PortExport = React.memo(Port)
-// $FlowFixMe
-PortExport.styles = styles
-
-export default PortExport
+export default (React.memo(Port): ComponentType<Props>)
```

Comment:
General rule for things like `memo` or `forwardRef` is that the exporter has to let others know what kind of thing they're importing. `ComponentType<Props>` is fine for most cases.